prime-router/settings/prod/0002-pa-philadelphia.yml

```
@@ -0,0 +1,49 @@
+---
+#
+# Organizations
+#
+- name: pa-phd
+  description: Pennsylvania Department of Health
+  jurisdiction: STATE
+  stateCode: PA
+  receivers:
+    - name: elr-philadelphia-prod
+      organizationName: pa-phd
+      topic: covid-19
+      jurisdictionalFilter: [ "matches(ordering_facility_state, PA)", "matches(ordering_facility_county, Philadelphia)" ]
+      deidentify: false
+      translation:
+        type: CUSTOM
+        schemaName: pa/pa-covid-19-redox
+        defaults:
+          processing_mode_code: P
+          redox_destination_id: 09261a90-bc55-4a88-953c-eff0240feab1
+          redox_destination_name: "CDC Philadelphia PDH Destination (p)"
+          redox_source_id: a3834c9d-e3ff-4602-b360-dcb0ad08fce5
+          redox_source_name: "Prime Data Hub (Prod)"
+        format: REDOX
+      transport:
+        type: REDOX
+        apiKey: 9e7f9bd1-a876-4f10-9d17-0833dc7bf60d # Redox prod
+
+- name: pa-philadelphia-phd
+  description: Philadelphia Department of Public Health
+  jurisdiction: COUNTY
+  stateCode: CO
```

```suggestion
  stateCode: PA
```

And it looks like Chester and Montgomery are wrong as well already in Prod.